core/kazoo_config/src/kapps_config.erl

```diff
@@ -853,7 +853,7 @@ migrate_config_setting(UpdatedFrom, Removed, To) ->
     case add_config_setting(To, Removed) of
         {'ok', UpdatedTo} ->
             {'ok', _} = kz_datamgr:save_doc(?KZ_CONFIG_DB, UpdatedTo),
-            {'ok', _} = kz_datamgr:save_doc(?KZ_CONFIG_DB, UpdatedFrom),
+            {'ok', _} = kz_datamgr:ensure_saved(?KZ_CONFIG_DB, UpdatedFrom),
```

Comment:
what tests did you make ?
client 1 -> get
client 2 -> get
client 1 -> set
client 2 -> set
value set from client 1 is lost